www/artists/emmalake/template.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="css/jackshadbolt.css">
+        <link rel="stylesheet" href="style.css" type="text/css">
+        <title>
+            Emma Lake Artist's Website
+        </title>
+        <link rel="stylesheet" href="style.css" type="text/css">
+        <link href='http://fonts.googleapis.com/css?family=Berkshire+Swash' rel='stylesheet' type='text/css'>
+        <style type="text/css">
+        p.c4 {clear: both;}
+        img.c3 {float: left; width: 30%; margin-right: 1%; margin-bottom: 0.5em;}
+        div.c2 {text-align: center}
+        img.c1 {float: left; width: 30%; margin-left: 5%; margin-right: 1%; margin-bottom: 0.5em;}
+        </style>
+    </head>
+    <body>
+        <div id="wrapper">
+            <div id="header">
+                <h1>
+                    Emma Lake Artists' Workshops
+                </h1>
+            </div>
+            <div id="navigation">
+                <a class="a" href="index.html">About the Workshops</a> <a class="a" href="artists.html">Artists Involved</a> <a class="a" href="gallery.html">Gallery</a>
+                <div id="content">
+                    <h1 class="jack">
+                        Name
+                    </h1>
+                    <br>
+                    <br>
+                    <br>
+                    <br>
+                    <br>
+                    <br>
+                    <div class="c2">
+                        <img class="image c1" src="img/.jpg" alt="no alt text">
+                    </div>
+                    <div class="c2">
+                        <img class="image c3" src="img/.jpg" alt="no alt text">
```

Comment:
While I appreciate you adding a template for the individual artist pages, as new artists may need to be added on as Emma Lake goes on, these img tags make me nervous. AJ is working on a file which confirms that all sources actually exist, and I'm 90% certain these two don't. I recommend adding dummy images instead.